### PR TITLE
Transmute and Calcify Fixes

### LINKED ISSUE #404: Monk Transmute
Forwarded by Andraste. Monk transmute apparently is having some issues so will need to have a look at it.

### PR DESCRIPTION
**Pull request by DazedxJoker**
Fixes #404, and adds calcifiedtorso and calcifiedskull to the simple and curing functions

### CODE REVIEW

File: svo (actions dictionary).xml [hunk 1]
```diff
@@ -13167,24 +13167,14 @@ if svo.me.class == 'Druid' then
...
-    transmutereps = 0,
     physical = {
       balanceless_act = true,
 
       isadvisable = function()
-        return (conf.transmute ~= 'none' and not defc.dragonform and (stats.currenthealth &lt; sys.transmuteamount or (sk.gettingfullstats and stats.currenthealth &lt; stats.maxhealth)) and not svo.doingaction'healhealth' and not svo.doingaction'transmute' and not codepaste.balanceful_codepaste() and svo.can_usemana() and (not affs.prone or svo.doingaction'prone') and not svo.dict.transmute.transmutespam) or false
+        return (conf.transmute ~= 'none' and not defc.dragonform and (stats.currenthealth &lt; sys.transmuteamount or (sk.gettingfullstats and stats.currenthealth &lt; stats.maxhealth)) and not svo.doingaction'healhealth' and not svo.doingaction'transmute' and not codepaste.balanceful_codepaste() and svo.can_usemana() and (not affs.prone or svo.doingaction'prone')) or false
       end,
 
       oncompleted = function()
```
Comment (vadi2) [inline svo (actions dictionary).xml]: Just remove it entirely

Comment (DazedxJoker) [reply svo (actions dictionary).xml]: Just remove transmutespam entirely? I thought I did. Or are you saying to remove the dict entry for transmute?

Comment (vadi2) [reply svo (actions dictionary).xml]: It looked like you left an empty `oncompleted` function in?

Comment (DazedxJoker) [reply svo (actions dictionary).xml]: Ah, missed it

**Pull request merged**